test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -378,6 +378,8 @@ describe('View existing record,', function() {
                         return browser.driver.getCurrentUrl();
                     }).then(function(url) {
                         expect(url.indexOf('recordset')).toBeGreaterThan(-1);
+                        // View more should not redirect to Search app
```

Comment:
This test is unnecessary. The line above tests exactly this case. If it's anything but `recordset` the above expectation fails. So if it's search, it will fail.